### DISCUSSION TITLE
Can you add npm (node package support)

### DISCUSSION THREAD

**Discussion by frog-o** [pyscript/pyscript#1251] [Proposals]
I don't know if this is doable but I think it would be since pyscript is supposed to handle JavaScript also. One of the nicest things about python is packages ; JavaScript also has packages with npm(although it's server side) could you allow for a way to use JavaScript packages from there.  What i would like to do is use code mirrors with pyscript.

**Comment by marimeireles** [comment]
Hey @frog-o this is being tracked [here](https://github.com/pyscript/pyscript/issues/675). I'm not sure how things have changed, tbh. Maybe new folks have new opinions about it. I'll bring it up in the discord chat :)
I guess I'll close this discussion as it's a dup of the issue.
